src/components/v5/common/ActionSidebar/partials/ActionSidebarContent/partials/MultiSigMembersError/MultiSigMembersError.tsx

```diff
@@ -27,13 +30,29 @@ export const MultiSigMembersError: FC<MultiSigMembersErrorProps> = ({
   updateCanCreateAction,
 }) => {
   const { watch } = useFormContext();
-  const [selectedAction, createdIn] = watch([
+  const [selectedAction, fromDomain, team] = watch([
     ACTION_TYPE_FIELD_NAME,
-    CREATED_IN_FIELD_NAME,
+    FROM_FIELD_NAME,
+    TEAM_FIELD_NAME,
   ]);
 
+  const domainId = useMemo(() => {
+    // for both of these actions you need roles in the parent domain
+    if (
+      selectedAction === Action.TransferFunds ||
+      selectedAction === Action.ManagePermissions
+    ) {
+      return Id.RootDomain;
+    }
+
+    return fromDomain ?? team ?? Id.RootDomain;
```

Comment:
I'm not sure if `Id.RootDomain` if no team is selected is ideal. This means if there are insufficient members in the root domain the "Not enough members" message will be shown before selecting a team, even if there are sub-domains where the threshold is met.

This should instead default to `undefined` if it doesn't break `useHasEnoughMembersWithPermissions`. Or prevent this message from showing until `fromDomain` has a value.

Reply:
Isn't this how it functioned before? `createdIn` was always set to the root team by default, that's why this just worked. But I agree that we shouldn't show the error until we check the domain, I'll try and handle this here, thanks for the heads up!

Reply:
As agreed on Discord, we're gonna be changing the error message to:
`There are not enough members in this or higher teams with the required permissions to approve this action. Select a different team, or assign more users with the required multi-sig permissions.` and implemented [here](https://github.com/JoinColony/colonyCDapp/blob/d267e29e1dccde7c0804568baf99ceba8c009093/src/i18n/en.json#L1091)